robolectric/src/main/java/org/robolectric/RobolectricTestRunner.java

```diff
@@ -330,7 +339,7 @@ protected AndroidManifest getAppManifest(Config config) {
       AndroidManifest appManifest;
       appManifest = appManifestsByFile.get(identifier);
```

Comment:
Do you need to add the package name to the identifier (which is used for caching)?